test/src/sdk-harness/test_projects/low_level_call/mod.rs

```diff
@@ -0,0 +1,178 @@
+use fuels::{
+    prelude::*,
+    tx::ContractId,
+    tx::{Bytes32, Input, Output, TxPointer, UtxoId},
+};
+
+macro_rules! fn_selector {
+    ( $fn_name: ident ( $($fn_arg: ty),* )  ) => {
+         ::fuels::core::code_gen::function_selector::resolve_fn_selector(stringify!($fn_name), &[$( <$fn_arg as ::fuels::core::Parameterize>::param_type() ),*]).to_vec()
+    }
+}
+macro_rules! calldata {
+    ( $($arg: expr),* ) => {
+        ::fuels::core::abi_encoder::ABIEncoder::encode(&[$(::fuels::core::Tokenizable::into_token($arg)),*]).unwrap().resolve(0)
+    }
+}
```

Comment:
Are these macros going to be no longer needed once there's better SDK support for these operations?

Reply:
These are actually super convenient in their own right (shout out to @segfault-magnet ). I wonder if they merit being added to the SDK as helper functions? 

Once we have full payload generation in the SDK, they _probably_ won't be necessary.